angular-frontend/README.md

```diff
@@ -115,6 +115,11 @@ tanzu apps workload apply \
 Note: change the namespace to where you would like to deploy this workload. Also define the (private) image registry you
 are allowed to push the source-image, like: `docker.io/username/repository`.
 
+--- StartAuthorization
+### Authorization
+If you want to use Tanzu's AppSSO Authorization server, please see `AUTHORIZATION.md for details.`
```

Comment:
**[nit]**: backticks should be closed on `AUTHORIZATION.md` only (no extra text)